### PR TITLE
new section to avoid hoarding

### PR DESCRIPTION
**Pull request by mellybelly**

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
I added a header

**Comment by nicolevasilevsky** [comment]
there is already a section on 'do not hoard masks'. Should these two be combined?

@nditada @jmcmurry

**Comment by mverzilli** [comment]
There's a lot of controversy around the masks topic in general, issues and PRs sprouting everywhere. I'm not sure how it ranks in prio against other content, but I have a feeling this topic would benefit from some first class attention from the scientific board. I've been trying to channel the contribution attempts but things move faster than I can react and I'm afraid I might be frustrating collaborators a bit.

**Comment by jaysonvirissimo** [comment]
I'm worried that, as is, there is tension between this and the "Stock up on food and essentials" section. On the one hand, we want people to make fewer trips to the store to limit their exposure, but if they are only buying what they "need" (a vague and ambiguous term), then doesn't that imply going more often and exposing themselves and others more than necessary?

To mean "stock up" and "hoard" sound denotatively similar, but with very different connotations. I think concrete examples of what counts as stocking up versus hoarding may help here.

**Comment by jmcmurry** [comment]
You are right that there is tension. When I wrote this March 3, no one was stocking up on anything - they were boarding planes and packing bars. Given where we are we may need to rethink this section. I think we need to say something like, if you're in an area that is not yet heavily impacted, now is the time to stock up as much as you can. If you are in an area that is heavily impacted, still try to minimize trips, but also be ready to share with those in need in your community. This is a very delicate balance.

File: _sections/act_and_prepare/en/03-dont/14-do-not-hoard.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{:.dont}
+
+### Do not hoard supplies 
+
+Items can easily become in short supply due to hoarding.  Common products experiencing shortages include toilet paper, bottled water, and medical supplies.
+
+Avoid purchasing large quantities of products, particularly those in short supply.  Get what you need. Leave the rest for others who need it.  If you have a large quantifiy of masks or other supplies consider donating it to groups in need such as healthcare providers or a local food pantry.
```
Comment (jmcmurry) [inline _sections/act_and_prepare/en/03-dont/14-do-not-hoard.md]:
```suggestion
Avoid purchasing large quantities of products, particularly those in short supply.  Get what you need for 2-4 weeks. Leave the rest for others who need it.  If you have a large quantity of masks or other supplies consider donating it to groups in need such as healthcare providers or a local food pantry.
```

**Pull request merged**